ledger/block/src/lib.rs

```diff
@@ -707,7 +707,7 @@ pub mod test_helpers {
         (block, transaction, private_key)
     }
 
-    pub(crate) fn sample_metadata(rng: &mut TestRng) -> Metadata<CurrentNetwork> {
+    pub(crate) fn sample_metadata() -> Metadata<CurrentNetwork> {
```

Comment:
note: while the fix was applicable to values beyond `u64::MAX`, we could introduce some randomness to this function to have greater coverage